src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_general.html.twig

```diff
@@ -87,6 +87,13 @@
             {{ form_widget(generalForm.default_status) }}
           </div>
         </div>
+        <div class="form-group row">
+          {{ ps.label_with_help(('Default specific price priorities'|trans), ('Specific prices are applied based on this priority list to all products unless product has its own specific list configured.'|trans({}, 'Admin.Shopparameters.Help'))) }}
+          <div class="col-sm">
+            {{ form_errors(generalForm.specific_price_priorities) }}
+            {{ form_widget(generalForm.specific_price_priorities) }}
```

Comment:
for some reason there is still empty label div rendered which looks terrible (even though label = false in type). Not focusing on styling in this PR, but if there anything simple I could do..

Reply:
In the recent ui kit yu can use aither `false` or `null` One displays an empty column and the other removes the label column completely

Though I don't think it's usable with the old form theme, but as I suggested in another comment I think it would be worth migrating this form and adding this new configuration field in a separate PR